end_to_end/mistral_load_and_test.sh

```diff
@@ -0,0 +1,18 @@
+set -e
+idx=$(date +%Y-%m-%d-%H-%M)
+
+export base_ckpt_path=gs://maxtext-external/mistral-7B-v0.1-maxtext/0/default
+export M_ENABLE_CHECKPOINTING=true
+export M_BASE_OUTPUT_DIRECTORY=gs://runner-maxtext-logs
+export M_DATASET_PATH=gs://maxtext-dataset
+export M_ASYNC_CHECKPOINTING=false
+
+#pip install torch
+#wget -P /tmp https://files.mistral-7b-v0-1.mistral.ai/mistral-7B-v0.1.tar
```

Comment:
Look we don't want the wget -- it would be best to use our own mirror. But otherwise we want to generate this checkpoint. The promise to the customer is we can generate this checkpoint!

Reply:
Will discuss offline for action item here. Thanks Rafi!